includes/polling/ports.inc.php

```diff
@@ -199,7 +199,8 @@
             $port_stats = snmpwalk_cache_oid($device, 'ifXEntry', $port_stats, 'IF-MIB');
         }
         $hc_test = array_slice($port_stats, 0, 1);
-        if (!isset($hc_test[0]['ifHCInOctets']) && !is_numeric($hc_test[0]['ifHCInOctets'])) {
+        if ((!isset($hc_test[0]['ifHCInOctets']) && !is_numeric($hc_test[0]['ifHCInOctets'])) &&
```

Comment:
Are you sure you don't want `||` here?  When you use or, it fixes devices missing ifHighSpeed...